tests/hil/tests/connection/test_connection.py

```diff
@@ -1,11 +1,28 @@
 import pytest
+import trio
+
+from golioth import RPCTimeout
 
 pytestmark = pytest.mark.anyio
 
+
 async def test_connect(board, device):
     # Set Golioth credentials
     golioth_cred = (await device.credentials.list())[0]
     board.set_golioth_psk_credentials(golioth_cred.identity, golioth_cred.key)
 
     # Confirm connection to Golioth
     assert None != board.wait_for_regex_in_line('Golioth CoAP client connected', timeout_s=120)
+
+
+async def test_stop_temporarily(board, device):
+    await trio.sleep(1)
+
+    try:
+        await device.rpc.stop_temporarily(10)
```

Comment:
Do we need the RPC if we have the start/stop in a loop? If we can manage it, would be nice to keep this test focused just on connections for isolation of testing.

Reply:
Initially I wanted to control that from outside. Unfortunately we don't have generic (platform agnostic) way of running commands over serial (shell), so I went with RPC instead. But overall it is better to keep tested functionalities separate and do not use RPC if not needed.

Followed your suggestion about periodic disconnects and seems to work well for this testcase.